locales/de/index.yml

```diff
@@ -1008,7 +1008,28 @@ wallet:
     success:
       title: "Die Karte wurde hinzugefügt!"
       continueButton: "Weiter"
-    failure: "Fehler beim Hinzufügen einer Karte"
+    failure:
+      GENERIC_ERROR:
+        title: Si è verificato un errore imprevisto
+        subtitle: Riprova o contatta l'assistenza
+        primaryAction: Chiudi
+      AUTH_ERROR:
```

Comment:
since DE locale is not mandatory, I'm pretty sure we shouldn't add keys unless they are translated.
can you confirm @thisisjp ?

Reply:
correct, it's also easier for our contributors to check the missing strings